system/Test/CIUnitTestCase.php

```diff
@@ -36,13 +36,17 @@
  * @filesource
  */
 
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase;
 use CodeIgniter\Log\TestLogger;
 
+if (! class_exists(TestCase::class)) {
+    class_alias(\PHPUnit_Framework_TestCase::class, TestCase::class);
+}
```

Comment:
upgrading to phpunit 6 will allow to not doing class_alias hack. Just use new `PHPUnit\Framework\TestCase`